doc/design.md

```diff
@@ -398,7 +398,7 @@ Needs: impl
 #### Retrieve GitHub Release Header from Release Letter
 `dsn~retrieve-github-release-header-from-release-letter~1`
```

Comment:
```suggestion
`dsn~retrieve-github-release-header-from-release-letter~2`
```
And update the dependent requirements of course.

Reply:
wilco next commit